### PR TITLE
Create a way to separate the Admin build from Minerva

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-565

**Description:**

1) Create makefile - a file which is used for creating multiple builds using make inbuilt automation tool in CLI
2) create two build folders with name Minerva and Admin inside Dist folder.

**Testing:**

`npm run test:unit`

 ## Screenshots:
(if applicable)

### CODE REVIEW

**Comment by FarajiA** [comment]
Ok this looks good, where is the output location though, can we configure that?

File: projects/admin/src/app/app.module.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ const providers = [AuthGuardService]
   ],
   imports: [
     AppRoutingModule,
+    BrowserModule
    // SharedModule
```
Comment (FarajiA) [inline projects/admin/src/app/app.module.ts:17]: We can remove.

**Review by FarajiA — CHANGES_REQUESTED** [review]
Ok just saw your Jira update that answers my questions from above:

> After that we have one single command which will create two separate folders inside dist/
i.e: dist/admin and dist/minerva

Can you add build instructions in the _README_ and then add NPM scripts to execute this? I don't think our current build process in NPM reflects this.

File: projects/admin/src/app/app.module.ts (hunk 1 above)
Comment (lovedeep92) [reply projects/admin/src/app/app.module.ts]: done !!

**Comment by lovedeep92** [comment]
@FarajiA , yup and its done!!

**Review by FarajiA — APPROVED** [review]
LGTM

**Pull request merged**